tests/src/tracing/tracevalidation/inducedgc/InducedGC.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.IO;
+using Tracing.Tests.Common;
+using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Parsers;
+using Microsoft.Diagnostics.Tracing.Parsers.Clr;
+
+namespace Tracing.Tests
+{
+    public static class TraceValidationInducedGC
+    {
+        private static int InducedGCIterations = 10;
+
+        public static int Main(string[] args)
+        {
+            bool pass = true;
+            bool keepOutput = false;
+
+            // Use the first arg as an output filename if there is one.
+            string outputFilename = null;
+            if (args.Length >= 1)
+            {
+                outputFilename = args[0];
+                keepOutput = true;
+            }
+            else
+            {
+                outputFilename = System.IO.Path.GetTempPath() + Guid.NewGuid().ToString() + ".netperf";
+            }
+
+            try
+            {
+                Console.WriteLine("\tStart: Enable tracing.");
+                TraceControl.EnableDefault(outputFilename);
+                Console.WriteLine("\tEnd: Enable tracing.\n");
+
+                Console.WriteLine("\tStart: Generate some events.");
+                for(int i=0; i<InducedGCIterations; i++)
+                {
+                    GC.Collect();
```

Comment:
to be sure that this call always induces the GC I have checked the docs. It [does](https://msdn.microsoft.com/en-us/library/xe0c2357(v=vs.110).aspx), always

Reply:
Yup, that is my understanding as well.  Thanks.